app/tests/cases_tests/test_api.py

```diff
@@ -212,6 +212,31 @@ def test_filter_images_api_view(client):
     assert response.json()["results"][0]["pk"] == str(im.pk)
 
 
+@pytest.mark.django_db
+def test_filter_reader_study_images_api_view(client):
+    rs = ReaderStudyFactory()
+    user = UserFactory()
+    rs.add_editor(user)
+
+    ds = DisplaySetFactory(reader_study=rs)
+
+    im = ImageFactory()
+    civ = ComponentInterfaceValueFactory(image=im)
+    with capture_on_commit_callbacks(execute=True):
+        ds.values.add(civ)
+
+    response = get_view_for_user(
+        client=client,
+        user=user,
+        viewname="api:image-list",
+        data={"reader_study": str(rs.pk)},
+        content_type="application/json",
+    )
+    assert response.status_code == 200
+    assert response.json()["count"] == 1
```

Comment:
Don't you need to add another image to ensure that it is filtered away?

Reply:
Definitely :slightly_smiling_face: